internal/pkg/application/decoder/payload/payload.go

```diff
@@ -73,6 +73,7 @@ func (p *PayloadImpl) Status() StatusImpl {
 	}
 	return StatusImpl{}
 }
+/*
 func (p *PayloadImpl) Measurements() []any {
```

Comment:
is this supposed to be commented out still?

Reply:
nope